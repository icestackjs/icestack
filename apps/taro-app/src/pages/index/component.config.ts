export default definePageConfig({
  navigationBarTitleText: '组件',
  navigationStyle: 'custom',
})
