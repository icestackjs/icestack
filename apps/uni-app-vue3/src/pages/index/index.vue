<template>
  <view class="content">
    <WeappTailwindcss></WeappTailwindcss>

    <view class="text-gray-900/50 mb-2 before:content-['当前系统主题:']">
      {{ themeRef }}
    </view>
    <view class="space-y-[20rpx] flex flex-col items-center">
      <view
        class="bg-[#010101] h-16 w-16 rounded-[20rpx] text-white flex justify-center items-center after:content-['hover']"
        hover-class="bg-[gray] after:!content-['good!']"></view>
      <view
        class="grid grid-cols-3 divide-x-[10px] divide-[#010101] divide-solid">
        <div :class="classArray">1</div>
        <div>2</div>
        <div :class="classArray">3</div>
      </view>
      <view
        class="w-32 py-2 rounded-md font-semibold text-white bg-pink-500 ring-4 ring-pink-300">
        Default
      </view>
      <view>
        <button class="text-[#fff]" :class="buttonClass" @click="increment">
          click here to inc {{ count }}
        </button>
      </view>

      <view class="test">@apply</view>
    </view>
  </view>
</template>

<script setup lang="ts">
import WeappTailwindcss from '@/components/WeappTailwindcss.vue'
import { useCounterStore } from '@/stores/counter'

const store = useCounterStore()
const { count } = storeToRefs(store)
const { increment } = store
const buttonColors = [
  'bg-[#000]',
  'bg-[#111]',
  'bg-[#222]',
  'bg-[#333]',
  'bg-[#444]',
  'bg-[#555]',
  'bg-[#666]',
  'bg-[#777]',
  'bg-[#888]',
  'bg-[#999]',
  'bg-[#aaa]',
  'bg-[#bbb]',
  'bg-[#ccc]',
  'bg-[#ddd]',
  'bg-[#eee]',
  'bg-[#fff]'
]
const title = ref('Hello')
const themeRef = ref(uni.getSystemInfoSync().theme)
const classArray = computed(() => [
  title.value ? 'bg-[#ff00ff]' : undefined,
  {
    'text-[#00ffff]': Boolean(title),
    "bg-[url('https://xxx.com/xx.webp')]": true,
    "bg-[url('https://yyyy.com/ccc.webp')]": true
  }
])
const buttonClass = computed(() => {
  return buttonColors[count.value % buttonColors.length]
})
// #ifdef MP
uni.onThemeChange(({ theme }: { theme: 'dark' | 'light' }) => {
  themeRef.value = theme
})
// #endif
onBeforeUnmount(() => {
  // #ifdef MP
  uni.offThemeChange(() => {
    console.log('offThemeChange')
  })
  // #endif
})

onLoad(() => {
  console.log('欢迎使用uni-app-vite-vue3-tailwindcss模板')
})
</script>

<style lang="scss" scoped>
.logo {
  @apply h-[100rpx] w-[100rpx];
}

.content {
  @apply flex flex-col items-center;
}

.test {
  @apply flex items-center justify-center h-[100px] w-[100px] rounded-[40px] bg-[#123456] bg-opacity-[0.54] text-[#ffffff] #{!important};
}
</style>
