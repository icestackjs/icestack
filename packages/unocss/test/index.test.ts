import path from 'node:path'
import ci from 'ci-info'
import { loadPresetOptions } from '@/index'
const loadDirectory = path.resolve(__dirname, '../my-ui')
describe.skipIf(ci.isCI)('index', () => {
  it('loadPresetOptions default', () => {
    const options = loadPresetOptions({
      loadDirectory,
      loadConfig: true
    })
    expect(options).toMatchSnapshot()
  })
})
