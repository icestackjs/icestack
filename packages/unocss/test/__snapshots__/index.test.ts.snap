// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`index > loadPresetOptions default 1`] = `
{
  "preflights": [
    {
      "getCSS": [Function],
    },
    {
      "getCSS": [Function],
    },
  ],
  "rules": [
    [
      /\\^alert\\$/,
      [Function],
    ],
    [
      /\\^alert-primary\\$/,
      [Function],
    ],
    [
      /\\^alert-success\\$/,
      [Function],
    ],
    [
      /\\^alert-warning\\$/,
      [Function],
    ],
    [
      /\\^alert-error\\$/,
      [Function],
    ],
    [
      /\\^alert-neutral\\$/,
      [Function],
    ],
    [
      /\\^avatar\\$/,
      [Function],
    ],
    [
      /\\^placeholder\\$/,
      [Function],
    ],
    [
      /\\^avatar-group\\$/,
      [Function],
    ],
    [
      /\\^badge\\$/,
      [Function],
    ],
    [
      /\\^badge-primary\\$/,
      [Function],
    ],
    [
      /\\^badge-success\\$/,
      [Function],
    ],
    [
      /\\^badge-warning\\$/,
      [Function],
    ],
    [
      /\\^badge-error\\$/,
      [Function],
    ],
    [
      /\\^badge-neutral\\$/,
      [Function],
    ],
    [
      /\\^badge-ghost\\$/,
      [Function],
    ],
    [
      /\\^badge-outline\\$/,
      [Function],
    ],
    [
      /\\^badge-xs\\$/,
      [Function],
    ],
    [
      /\\^badge-sm\\$/,
      [Function],
    ],
    [
      /\\^badge-md\\$/,
      [Function],
    ],
    [
      /\\^badge-lg\\$/,
      [Function],
    ],
    [
      /\\^btn\\$/,
      [Function],
    ],
    [
      /\\^btn-disabled\\$/,
      [Function],
    ],
    [
      /\\^btn-focus-visible\\$/,
      [Function],
    ],
    [
      /\\^btn-hover\\$/,
      [Function],
    ],
    [
      /\\^btn-active\\$/,
      [Function],
    ],
    [
      /\\^btn-primary\\$/,
      [Function],
    ],
    [
      /\\^btn-success\\$/,
      [Function],
    ],
    [
      /\\^btn-warning\\$/,
      [Function],
    ],
    [
      /\\^btn-error\\$/,
      [Function],
    ],
    [
      /\\^btn-neutral\\$/,
      [Function],
    ],
    [
      /\\^glass\\$/,
      [Function],
    ],
    [
      /\\^btn-ghost\\$/,
      [Function],
    ],
    [
      /\\^btn-link\\$/,
      [Function],
    ],
    [
      /\\^btn-outline\\$/,
      [Function],
    ],
    [
      /\\^btn-xs\\$/,
      [Function],
    ],
    [
      /\\^btn-sm\\$/,
      [Function],
    ],
    [
      /\\^btn-md\\$/,
      [Function],
    ],
    [
      /\\^btn-lg\\$/,
      [Function],
    ],
    [
      /\\^btn-wide\\$/,
      [Function],
    ],
    [
      /\\^btn-block\\$/,
      [Function],
    ],
    [
      /\\^btn-square\\$/,
      [Function],
    ],
    [
      /\\^btn-circle\\$/,
      [Function],
    ],
    [
      /\\^chat\\$/,
      [Function],
    ],
    [
      /\\^chat-image\\$/,
      [Function],
    ],
    [
      /\\^chat-header\\$/,
      [Function],
    ],
    [
      /\\^chat-footer\\$/,
      [Function],
    ],
    [
      /\\^chat-bubble\\$/,
      [Function],
    ],
    [
      /\\^chat-start\\$/,
      [Function],
    ],
    [
      /\\^chat-end\\$/,
      [Function],
    ],
    [
      /\\^chat-bubble-primary\\$/,
      [Function],
    ],
    [
      /\\^chat-bubble-success\\$/,
      [Function],
    ],
    [
      /\\^chat-bubble-warning\\$/,
      [Function],
    ],
    [
      /\\^chat-bubble-error\\$/,
      [Function],
    ],
    [
      /\\^chat-bubble-neutral\\$/,
      [Function],
    ],
    [
      /\\^checkbox\\$/,
      [Function],
    ],
    [
      /\\^checkbox-primary\\$/,
      [Function],
    ],
    [
      /\\^checkbox-success\\$/,
      [Function],
    ],
    [
      /\\^checkbox-warning\\$/,
      [Function],
    ],
    [
      /\\^checkbox-error\\$/,
      [Function],
    ],
    [
      /\\^checkbox-neutral\\$/,
      [Function],
    ],
    [
      /\\^checkbox-xs\\$/,
      [Function],
    ],
    [
      /\\^checkbox-sm\\$/,
      [Function],
    ],
    [
      /\\^checkbox-md\\$/,
      [Function],
    ],
    [
      /\\^checkbox-lg\\$/,
      [Function],
    ],
    [
      /\\^input\\$/,
      [Function],
    ],
    [
      /\\^input-bordered\\$/,
      [Function],
    ],
    [
      /\\^input-ghost\\$/,
      [Function],
    ],
    [
      /\\^input-primary\\$/,
      [Function],
    ],
    [
      /\\^input-success\\$/,
      [Function],
    ],
    [
      /\\^input-warning\\$/,
      [Function],
    ],
    [
      /\\^input-error\\$/,
      [Function],
    ],
    [
      /\\^input-neutral\\$/,
      [Function],
    ],
    [
      /\\^input-disabled\\$/,
      [Function],
    ],
    [
      /\\^input-xs\\$/,
      [Function],
    ],
    [
      /\\^input-sm\\$/,
      [Function],
    ],
    [
      /\\^input-md\\$/,
      [Function],
    ],
    [
      /\\^input-lg\\$/,
      [Function],
    ],
    [
      /\\^link\\$/,
      [Function],
    ],
    [
      /\\^link-hover\\$/,
      [Function],
    ],
    [
      /\\^link-primary\\$/,
      [Function],
    ],
    [
      /\\^link-success\\$/,
      [Function],
    ],
    [
      /\\^link-warning\\$/,
      [Function],
    ],
    [
      /\\^link-error\\$/,
      [Function],
    ],
    [
      /\\^link-neutral\\$/,
      [Function],
    ],
    [
      /\\^progress\\$/,
      [Function],
    ],
    [
      /\\^progress-primary\\$/,
      [Function],
    ],
    [
      /\\^progress-success\\$/,
      [Function],
    ],
    [
      /\\^progress-warning\\$/,
      [Function],
    ],
    [
      /\\^progress-error\\$/,
      [Function],
    ],
    [
      /\\^progress-neutral\\$/,
      [Function],
    ],
    [
      /\\^radio\\$/,
      [Function],
    ],
    [
      /\\^radio-primary\\$/,
      [Function],
    ],
    [
      /\\^radio-success\\$/,
      [Function],
    ],
    [
      /\\^radio-warning\\$/,
      [Function],
    ],
    [
      /\\^radio-error\\$/,
      [Function],
    ],
    [
      /\\^radio-neutral\\$/,
      [Function],
    ],
    [
      /\\^radio-xs\\$/,
      [Function],
    ],
    [
      /\\^radio-sm\\$/,
      [Function],
    ],
    [
      /\\^radio-md\\$/,
      [Function],
    ],
    [
      /\\^radio-lg\\$/,
      [Function],
    ],
    [
      /\\^range\\$/,
      [Function],
    ],
    [
      /\\^range-primary\\$/,
      [Function],
    ],
    [
      /\\^range-success\\$/,
      [Function],
    ],
    [
      /\\^range-warning\\$/,
      [Function],
    ],
    [
      /\\^range-error\\$/,
      [Function],
    ],
    [
      /\\^range-neutral\\$/,
      [Function],
    ],
    [
      /\\^range-xs\\$/,
      [Function],
    ],
    [
      /\\^range-sm\\$/,
      [Function],
    ],
    [
      /\\^range-md\\$/,
      [Function],
    ],
    [
      /\\^range-lg\\$/,
      [Function],
    ],
    [
      /\\^select\\$/,
      [Function],
    ],
    [
      /\\^select-bordered\\$/,
      [Function],
    ],
    [
      /\\^select-ghost\\$/,
      [Function],
    ],
    [
      /\\^select-primary\\$/,
      [Function],
    ],
    [
      /\\^select-success\\$/,
      [Function],
    ],
    [
      /\\^select-warning\\$/,
      [Function],
    ],
    [
      /\\^select-error\\$/,
      [Function],
    ],
    [
      /\\^select-neutral\\$/,
      [Function],
    ],
    [
      /\\^select-disabled\\$/,
      [Function],
    ],
    [
      /\\^select-multiple\\$/,
      [Function],
    ],
    [
      /\\^select-md\\$/,
      [Function],
    ],
    [
      /\\^select-lg\\$/,
      [Function],
    ],
    [
      /\\^select-sm\\$/,
      [Function],
    ],
    [
      /\\^select-xs\\$/,
      [Function],
    ],
    [
      /\\^textarea\\$/,
      [Function],
    ],
    [
      /\\^textarea-bordered\\$/,
      [Function],
    ],
    [
      /\\^textarea-ghost\\$/,
      [Function],
    ],
    [
      /\\^textarea-primary\\$/,
      [Function],
    ],
    [
      /\\^textarea-success\\$/,
      [Function],
    ],
    [
      /\\^textarea-warning\\$/,
      [Function],
    ],
    [
      /\\^textarea-error\\$/,
      [Function],
    ],
    [
      /\\^textarea-neutral\\$/,
      [Function],
    ],
    [
      /\\^textarea-disabled\\$/,
      [Function],
    ],
    [
      /\\^textarea-xs\\$/,
      [Function],
    ],
    [
      /\\^textarea-sm\\$/,
      [Function],
    ],
    [
      /\\^textarea-md\\$/,
      [Function],
    ],
    [
      /\\^textarea-lg\\$/,
      [Function],
    ],
    [
      /\\^toggle\\$/,
      [Function],
    ],
    [
      /\\^toggle-primary\\$/,
      [Function],
    ],
    [
      /\\^toggle-success\\$/,
      [Function],
    ],
    [
      /\\^toggle-warning\\$/,
      [Function],
    ],
    [
      /\\^toggle-error\\$/,
      [Function],
    ],
    [
      /\\^toggle-neutral\\$/,
      [Function],
    ],
    [
      /\\^toggle-xs\\$/,
      [Function],
    ],
    [
      /\\^toggle-sm\\$/,
      [Function],
    ],
    [
      /\\^toggle-md\\$/,
      [Function],
    ],
    [
      /\\^toggle-lg\\$/,
      [Function],
    ],
    [
      /\\^loading\\$/,
      [Function],
    ],
    [
      /\\^loading-xs\\$/,
      [Function],
    ],
    [
      /\\^loading-sm\\$/,
      [Function],
    ],
    [
      /\\^loading-md\\$/,
      [Function],
    ],
    [
      /\\^loading-lg\\$/,
      [Function],
    ],
    [
      /\\^loading-audio\\$/,
      [Function],
    ],
    [
      /\\^loading-ball-triangle\\$/,
      [Function],
    ],
    [
      /\\^loading-bars\\$/,
      [Function],
    ],
    [
      /\\^loading-circles\\$/,
      [Function],
    ],
    [
      /\\^loading-grid\\$/,
      [Function],
    ],
    [
      /\\^loading-hearts\\$/,
      [Function],
    ],
    [
      /\\^loading-oval\\$/,
      [Function],
    ],
    [
      /\\^loading-puff\\$/,
      [Function],
    ],
    [
      /\\^loading-rings\\$/,
      [Function],
    ],
    [
      /\\^loading-spinning-circles\\$/,
      [Function],
    ],
    [
      /\\^loading-tail-spin\\$/,
      [Function],
    ],
    [
      /\\^loading-three-dots\\$/,
      [Function],
    ],
    [
      /\\^mask\\$/,
      [Function],
    ],
    [
      /\\^mask-half-1\\$/,
      [Function],
    ],
    [
      /\\^mask-half-2\\$/,
      [Function],
    ],
    [
      /\\^mask-squircle\\$/,
      [Function],
    ],
    [
      /\\^mask-decagon\\$/,
      [Function],
    ],
    [
      /\\^mask-diamond\\$/,
      [Function],
    ],
    [
      /\\^mask-heart\\$/,
      [Function],
    ],
    [
      /\\^mask-hexagon\\$/,
      [Function],
    ],
    [
      /\\^mask-hexagon-2\\$/,
      [Function],
    ],
    [
      /\\^mask-circle\\$/,
      [Function],
    ],
    [
      /\\^mask-parallelogram\\$/,
      [Function],
    ],
    [
      /\\^mask-parallelogram-2\\$/,
      [Function],
    ],
    [
      /\\^mask-parallelogram-3\\$/,
      [Function],
    ],
    [
      /\\^mask-parallelogram-4\\$/,
      [Function],
    ],
    [
      /\\^mask-pentagon\\$/,
      [Function],
    ],
    [
      /\\^mask-square\\$/,
      [Function],
    ],
    [
      /\\^mask-star\\$/,
      [Function],
    ],
    [
      /\\^mask-star-2\\$/,
      [Function],
    ],
    [
      /\\^mask-triangle\\$/,
      [Function],
    ],
    [
      /\\^mask-triangle-2\\$/,
      [Function],
    ],
    [
      /\\^mask-triangle-3\\$/,
      [Function],
    ],
    [
      /\\^mask-triangle-4\\$/,
      [Function],
    ],
    [
      /\\^table\\$/,
      [Function],
    ],
    [
      /\\^table-pin-rows\\$/,
      [Function],
    ],
    [
      /\\^table-pin-cols\\$/,
      [Function],
    ],
    [
      /\\^table-zebra\\$/,
      [Function],
    ],
    [
      /\\^active\\$/,
      [Function],
    ],
    [
      /\\^hover\\$/,
      [Function],
    ],
    [
      /\\^table-zebra-zebra\\$/,
      [Function],
    ],
    [
      /\\^table-xs\\$/,
      [Function],
    ],
    [
      /\\^table-sm\\$/,
      [Function],
    ],
    [
      /\\^table-md\\$/,
      [Function],
    ],
    [
      /\\^table-lg\\$/,
      [Function],
    ],
    [
      /\\^skeleton\\$/,
      [Function],
    ],
    [
      /\\^skeleton-title\\$/,
      [Function],
    ],
    [
      /\\^skeleton-paragraph\\$/,
      [Function],
    ],
    [
      /\\^skeleton-avatar\\$/,
      [Function],
    ],
    [
      /\\^form-control\\$/,
      [Function],
    ],
    [
      /\\^label\\$/,
      [Function],
    ],
    [
      /\\^label-text\\$/,
      [Function],
    ],
    [
      /\\^label-text-alt\\$/,
      [Function],
    ],
    [
      /\\^radial-progress\\$/,
      [Function],
    ],
    [
      /\\^countdown\\$/,
      [Function],
    ],
    [
      /\\^diff\\$/,
      [Function],
    ],
    [
      /\\^diff-resizer\\$/,
      [Function],
    ],
    [
      /\\^diff-item-1\\$/,
      [Function],
    ],
    [
      /\\^diff-item-2\\$/,
      [Function],
    ],
    [
      /\\^kbd\\$/,
      [Function],
    ],
    [
      /\\^kbd-xs\\$/,
      [Function],
    ],
    [
      /\\^kbd-sm\\$/,
      [Function],
    ],
    [
      /\\^kbd-md\\$/,
      [Function],
    ],
    [
      /\\^kbd-lg\\$/,
      [Function],
    ],
    [
      /\\^tooltip\\$/,
      [Function],
    ],
    [
      /\\^tooltip-top\\$/,
      [Function],
    ],
    [
      /\\^tooltip-bottom\\$/,
      [Function],
    ],
    [
      /\\^tooltip-left\\$/,
      [Function],
    ],
    [
      /\\^tooltip-right\\$/,
      [Function],
    ],
    [
      /\\^tooltip-primary\\$/,
      [Function],
    ],
    [
      /\\^tooltip-success\\$/,
      [Function],
    ],
    [
      /\\^tooltip-warning\\$/,
      [Function],
    ],
    [
      /\\^tooltip-error\\$/,
      [Function],
    ],
    [
      /\\^tooltip-neutral\\$/,
      [Function],
    ],
    [
      /\\^tooltip-open\\$/,
      [Function],
    ],
    [
      /\\^toast\\$/,
      [Function],
    ],
    [
      /\\^toast-start\\$/,
      [Function],
    ],
    [
      /\\^toast-center\\$/,
      [Function],
    ],
    [
      /\\^toast-end\\$/,
      [Function],
    ],
    [
      /\\^toast-bottom\\$/,
      [Function],
    ],
    [
      /\\^toast-middle\\$/,
      [Function],
    ],
    [
      /\\^toast-top\\$/,
      [Function],
    ],
    [
      /\\^steps\\$/,
      [Function],
    ],
    [
      /\\^step\\$/,
      [Function],
    ],
    [
      /\\^step-primary\\$/,
      [Function],
    ],
    [
      /\\^step-success\\$/,
      [Function],
    ],
    [
      /\\^step-warning\\$/,
      [Function],
    ],
    [
      /\\^step-error\\$/,
      [Function],
    ],
    [
      /\\^step-neutral\\$/,
      [Function],
    ],
    [
      /\\^steps-horizontal\\$/,
      [Function],
    ],
    [
      /\\^steps-vertical\\$/,
      [Function],
    ],
    [
      /\\^collapse\\$/,
      [Function],
    ],
    [
      /\\^collapse-title\\$/,
      [Function],
    ],
    [
      /\\^collapse-content\\$/,
      [Function],
    ],
    [
      /\\^collapse-open\\$/,
      [Function],
    ],
    [
      /\\^collapse-close\\$/,
      [Function],
    ],
    [
      /\\^collapse-arrow\\$/,
      [Function],
    ],
    [
      /\\^collapse-plus\\$/,
      [Function],
    ],
    [
      /\\^join\\$/,
      [Function],
    ],
    [
      /\\^join-item\\$/,
      [Function],
    ],
    [
      /\\^dropdown\\$/,
      [Function],
    ],
    [
      /\\^join-vertical\\$/,
      [Function],
    ],
    [
      /\\^join-horizontal\\$/,
      [Function],
    ],
    [
      /\\^indicator\\$/,
      [Function],
    ],
    [
      /\\^indicator-item\\$/,
      [Function],
    ],
    [
      /\\^indicator-start\\$/,
      [Function],
    ],
    [
      /\\^indicator-center\\$/,
      [Function],
    ],
    [
      /\\^indicator-end\\$/,
      [Function],
    ],
    [
      /\\^indicator-bottom\\$/,
      [Function],
    ],
    [
      /\\^indicator-middle\\$/,
      [Function],
    ],
    [
      /\\^indicator-top\\$/,
      [Function],
    ],
    [
      /\\^divider\\$/,
      [Function],
    ],
    [
      /\\^divider-start\\$/,
      [Function],
    ],
    [
      /\\^divider-end\\$/,
      [Function],
    ],
    [
      /\\^divider-primary\\$/,
      [Function],
    ],
    [
      /\\^divider-success\\$/,
      [Function],
    ],
    [
      /\\^divider-warning\\$/,
      [Function],
    ],
    [
      /\\^divider-error\\$/,
      [Function],
    ],
    [
      /\\^divider-neutral\\$/,
      [Function],
    ],
    [
      /\\^divider-horizontal\\$/,
      [Function],
    ],
    [
      /\\^divider-vertical\\$/,
      [Function],
    ],
    [
      /\\^stack\\$/,
      [Function],
    ],
    [
      /\\^tabs\\$/,
      [Function],
    ],
    [
      /\\^tabs-lifted\\$/,
      [Function],
    ],
    [
      /\\^tab-content\\$/,
      [Function],
    ],
    [
      /\\^tab\\$/,
      [Function],
    ],
    [
      /\\^tab-active\\$/,
      [Function],
    ],
    [
      /\\^tab-disabled\\$/,
      [Function],
    ],
    [
      /\\^tabs-bordered\\$/,
      [Function],
    ],
    [
      /\\^tabs-boxed\\$/,
      [Function],
    ],
    [
      /\\^tabs-md\\$/,
      [Function],
    ],
    [
      /\\^tabs-lg\\$/,
      [Function],
    ],
    [
      /\\^tabs-sm\\$/,
      [Function],
    ],
    [
      /\\^tabs-xs\\$/,
      [Function],
    ],
    [
      /\\^dropdown-content\\$/,
      [Function],
    ],
    [
      /\\^dropdown-end\\$/,
      [Function],
    ],
    [
      /\\^dropdown-left\\$/,
      [Function],
    ],
    [
      /\\^dropdown-right\\$/,
      [Function],
    ],
    [
      /\\^dropdown-bottom\\$/,
      [Function],
    ],
    [
      /\\^dropdown-top\\$/,
      [Function],
    ],
    [
      /\\^dropdown-open\\$/,
      [Function],
    ],
    [
      /\\^dropdown-hover\\$/,
      [Function],
    ],
    [
      /\\^swap\\$/,
      [Function],
    ],
    [
      /\\^swap-on\\$/,
      [Function],
    ],
    [
      /\\^swap-indeterminate\\$/,
      [Function],
    ],
    [
      /\\^swap-off\\$/,
      [Function],
    ],
    [
      /\\^swap-active\\$/,
      [Function],
    ],
    [
      /\\^swap-rotate\\$/,
      [Function],
    ],
    [
      /\\^swap-flip\\$/,
      [Function],
    ],
    [
      /\\^card\\$/,
      [Function],
    ],
    [
      /\\^card-body\\$/,
      [Function],
    ],
    [
      /\\^card-actions\\$/,
      [Function],
    ],
    [
      /\\^image-full\\$/,
      [Function],
    ],
    [
      /\\^bordered\\$/,
      [Function],
    ],
    [
      /\\^card-bordered\\$/,
      [Function],
    ],
    [
      /\\^compact\\$/,
      [Function],
    ],
    [
      /\\^card-title\\$/,
      [Function],
    ],
    [
      /\\^card-side\\$/,
      [Function],
    ],
    [
      /\\^card-compact\\$/,
      [Function],
    ],
    [
      /\\^card-normal\\$/,
      [Function],
    ],
    [
      /\\^carousel\\$/,
      [Function],
    ],
    [
      /\\^carousel-vertical\\$/,
      [Function],
    ],
    [
      /\\^carousel-item\\$/,
      [Function],
    ],
    [
      /\\^carousel-center\\$/,
      [Function],
    ],
    [
      /\\^carousel-end\\$/,
      [Function],
    ],
    [
      /\\^stats\\$/,
      [Function],
    ],
    [
      /\\^stat\\$/,
      [Function],
    ],
    [
      /\\^stat-figure\\$/,
      [Function],
    ],
    [
      /\\^stat-title\\$/,
      [Function],
    ],
    [
      /\\^stat-value\\$/,
      [Function],
    ],
    [
      /\\^stat-desc\\$/,
      [Function],
    ],
    [
      /\\^stat-actions\\$/,
      [Function],
    ],
    [
      /\\^stats-horizontal\\$/,
      [Function],
    ],
    [
      /\\^stats-vertical\\$/,
      [Function],
    ],
    [
      /\\^timeline\\$/,
      [Function],
    ],
    [
      /\\^timeline-start\\$/,
      [Function],
    ],
    [
      /\\^timeline-middle\\$/,
      [Function],
    ],
    [
      /\\^timeline-end\\$/,
      [Function],
    ],
    [
      /\\^timeline-box\\$/,
      [Function],
    ],
    [
      /\\^timeline-vertical\\$/,
      [Function],
    ],
    [
      /\\^timeline-compact\\$/,
      [Function],
    ],
    [
      /\\^timeline-horizontal\\$/,
      [Function],
    ],
    [
      /\\^timeline-snap-icon\\$/,
      [Function],
    ],
    [
      /\\^breadcrumbs\\$/,
      [Function],
    ],
    [
      /\\^btm-nav\\$/,
      [Function],
    ],
    [
      /\\^disabled\\$/,
      [Function],
    ],
    [
      /\\^btm-nav-xs\\$/,
      [Function],
    ],
    [
      /\\^btm-nav-label\\$/,
      [Function],
    ],
    [
      /\\^btm-nav-sm\\$/,
      [Function],
    ],
    [
      /\\^btm-nav-md\\$/,
      [Function],
    ],
    [
      /\\^btm-nav-lg\\$/,
      [Function],
    ],
    [
      /\\^menu\\$/,
      [Function],
    ],
    [
      /\\^menu-title\\$/,
      [Function],
    ],
    [
      /\\^menu-dropdown\\$/,
      [Function],
    ],
    [
      /\\^menu-dropdown-show\\$/,
      [Function],
    ],
    [
      /\\^focus\\$/,
      [Function],
    ],
    [
      /\\^menu-dropdown-toggle\\$/,
      [Function],
    ],
    [
      /\\^menu-horizontal\\$/,
      [Function],
    ],
    [
      /\\^menu-vertical\\$/,
      [Function],
    ],
    [
      /\\^menu-xs\\$/,
      [Function],
    ],
    [
      /\\^menu-sm\\$/,
      [Function],
    ],
    [
      /\\^menu-md\\$/,
      [Function],
    ],
    [
      /\\^menu-lg\\$/,
      [Function],
    ],
    [
      /\\^navbar\\$/,
      [Function],
    ],
    [
      /\\^navbar-start\\$/,
      [Function],
    ],
    [
      /\\^navbar-center\\$/,
      [Function],
    ],
    [
      /\\^navbar-end\\$/,
      [Function],
    ],
    [
      /\\^file-input\\$/,
      [Function],
    ],
    [
      /\\^file-input-bordered\\$/,
      [Function],
    ],
    [
      /\\^file-input-ghost\\$/,
      [Function],
    ],
    [
      /\\^file-input-primary\\$/,
      [Function],
    ],
    [
      /\\^file-input-success\\$/,
      [Function],
    ],
    [
      /\\^file-input-warning\\$/,
      [Function],
    ],
    [
      /\\^file-input-error\\$/,
      [Function],
    ],
    [
      /\\^file-input-neutral\\$/,
      [Function],
    ],
    [
      /\\^file-input-disabled\\$/,
      [Function],
    ],
    [
      /\\^file-input-xs\\$/,
      [Function],
    ],
    [
      /\\^file-input-sm\\$/,
      [Function],
    ],
    [
      /\\^file-input-md\\$/,
      [Function],
    ],
    [
      /\\^file-input-lg\\$/,
      [Function],
    ],
    [
      /\\^rating\\$/,
      [Function],
    ],
    [
      /\\^rating-hidden\\$/,
      [Function],
    ],
    [
      /\\^rating-half\\$/,
      [Function],
    ],
    [
      /\\^rating-xs\\$/,
      [Function],
    ],
    [
      /\\^rating-sm\\$/,
      [Function],
    ],
    [
      /\\^rating-md\\$/,
      [Function],
    ],
    [
      /\\^rating-lg\\$/,
      [Function],
    ],
    [
      /\\^drawer\\$/,
      [Function],
    ],
    [
      /\\^drawer-content\\$/,
      [Function],
    ],
    [
      /\\^drawer-side\\$/,
      [Function],
    ],
    [
      /\\^drawer-overlay\\$/,
      [Function],
    ],
    [
      /\\^drawer-toggle\\$/,
      [Function],
    ],
    [
      /\\^drawer-end\\$/,
      [Function],
    ],
    [
      /\\^drawer-button\\$/,
      [Function],
    ],
    [
      /\\^drawer-open\\$/,
      [Function],
    ],
    [
      /\\^footer\\$/,
      [Function],
    ],
    [
      /\\^footer-center\\$/,
      [Function],
    ],
    [
      /\\^footer-title\\$/,
      [Function],
    ],
    [
      /\\^hero\\$/,
      [Function],
    ],
    [
      /\\^hero-overlay\\$/,
      [Function],
    ],
    [
      /\\^hero-content\\$/,
      [Function],
    ],
    [
      /\\^no-animation\\$/,
      [Function],
    ],
    [
      /\\^tab-border-none\\$/,
      [Function],
    ],
    [
      /\\^tab-border\\$/,
      [Function],
    ],
    [
      /\\^tab-border-2\\$/,
      [Function],
    ],
    [
      /\\^tab-border-3\\$/,
      [Function],
    ],
    [
      /\\^tab-rounded-none\\$/,
      [Function],
    ],
    [
      /\\^tab-rounded-lg\\$/,
      [Function],
    ],
  ],
  "theme": {
    "colors": {
      "base-100": "rgba(var(--ice-base-100),<alpha-value>)",
      "base-1000": "rgba(var(--ice-base-1000),<alpha-value>)",
      "base-1100": "rgba(var(--ice-base-1100),<alpha-value>)",
      "base-1200": "rgba(var(--ice-base-1200),<alpha-value>)",
      "base-1300": "rgba(var(--ice-base-1300),<alpha-value>)",
      "base-200": "rgba(var(--ice-base-200),<alpha-value>)",
      "base-300": "rgba(var(--ice-base-300),<alpha-value>)",
      "base-400": "rgba(var(--ice-base-400),<alpha-value>)",
      "base-500": "rgba(var(--ice-base-500),<alpha-value>)",
      "base-600": "rgba(var(--ice-base-600),<alpha-value>)",
      "base-700": "rgba(var(--ice-base-700),<alpha-value>)",
      "base-800": "rgba(var(--ice-base-800),<alpha-value>)",
      "base-900": "rgba(var(--ice-base-900),<alpha-value>)",
      "base-content": "rgba(var(--ice-base-content),<alpha-value>)",
      "current": "currentColor",
      "error": "rgba(var(--ice-error),<alpha-value>)",
      "error-active": "rgba(var(--ice-error-active),<alpha-value>)",
      "error-content": "rgba(var(--ice-error-content),<alpha-value>)",
      "error-hover": "rgba(var(--ice-error-hover),<alpha-value>)",
      "neutral": "rgba(var(--ice-neutral),<alpha-value>)",
      "neutral-active": "rgba(var(--ice-neutral-active),<alpha-value>)",
      "neutral-content": "rgba(var(--ice-neutral-content),<alpha-value>)",
      "neutral-hover": "rgba(var(--ice-neutral-hover),<alpha-value>)",
      "primary": "rgba(var(--ice-primary),<alpha-value>)",
      "primary-active": "rgba(var(--ice-primary-active),<alpha-value>)",
      "primary-content": "rgba(var(--ice-primary-content),<alpha-value>)",
      "primary-hover": "rgba(var(--ice-primary-hover),<alpha-value>)",
      "success": "rgba(var(--ice-success),<alpha-value>)",
      "success-active": "rgba(var(--ice-success-active),<alpha-value>)",
      "success-content": "rgba(var(--ice-success-content),<alpha-value>)",
      "success-hover": "rgba(var(--ice-success-hover),<alpha-value>)",
      "transparent": "transparent",
      "warning": "rgba(var(--ice-warning),<alpha-value>)",
      "warning-active": "rgba(var(--ice-warning-active),<alpha-value>)",
      "warning-content": "rgba(var(--ice-warning-content),<alpha-value>)",
      "warning-hover": "rgba(var(--ice-warning-hover),<alpha-value>)",
    },
  },
}
`;
